src/app/loadout/LoadoutEditPopup.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { t } from 'i18next';
+import { getLoadoutClassDisplay } from './loadout.service';
+import './loadout-edit-popup.scss';
+
+interface Props {
+  loadoutClassType: number;
+  loadoutName: string;
+  changeNameHandler(): void;
+  editHandler(): void;
+}
+
+const getAlreadyExistsTranslation = (loadoutClassType: number) => {
```

Comment:
In the DIM codebase we generally prefer a function to an arrow declaration, so like:

```
function getAlreadyExistsTranslation(loadoutClassType: number) {
   ...
}
```